conan/tools/build/compiler.py

```diff
@@ -3,32 +3,29 @@
 
 
 def check_min_compiler_version(conanfile, compiler_restrictions):
-    """
-    Checks if the current compiler and its version meet the minimum requirements
-    specified in a list of restrictions.
+    """Checks if the current compiler and its version meet the minimum requirements.
```

Comment:
```suggestion
    """(Experimental) Checks if the current compiler and its version meet the minimum requirements.
```

I think it would be good to put at least an ``experimental`` or ``preview`` qualifier, as this is brand new, there might be something we might have missed in the ux/ui that we might want to improve?